### PR TITLE
Move join url in prow config

### PR DESCRIPTION
**Pull request by kargakis**
Bump and config update for https://github.com/kubernetes/test-infra/pull/5951

/assign cjwagner

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Dependent PR merged,

/lgtm

**Comment by cjwagner** [comment]
Please address https://github.com/kubernetes/test-infra/pull/5951#pullrequestreview-83591482 before merging this.

File: prow/plugins.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ triggers:
   - tensorflow/k8s
   - GoogleCloudPlatform/k8s-multicluster-ingress
   trusted_org: kubernetes
+  join_org_url: "https://github.com/kubernetes/community/blob/master/community-membership.md#member"
```
Comment (cblecker) [inline prow/plugins.yaml]: community nit: can we use `https://git.k8s.io/community/community-membership.md#member` for k8s, please?

**Pull request merged**